website/components/live-example/LiveExample.tsx

```diff
@@ -170,6 +172,9 @@ function LiveExample<ComponentProps extends ComponentPropsInterface>({
               {...sharedProps}
               options={knobConfig?.options as Array<string>}
               value={props[propName] as string}
+              // Disables `as` dropdown when hasHrefProp is true in order to accurately portray the effects of each respesctively.
+              // In our live examples, we give precedence to the hasHrefProp to demonstrate that passing an href to a Box component renders it as an anchor tag.
+              disabled={propName === 'as' && props.hasHrefProp ? true : false}
```

Comment:
this isn't the right place to define this. there should be a way to control this from the definition of the Box live example. there could be some other component that has both `as` and `href` props that are compatible, e.g. because the props are passed to different sub-components.

Reply:
Hm... I'll have to think about how to approach that then, as the information being passed to the knobs doesn't receive any information about current prop values.

Reply:
You're not wrong, I just have to rethink.

Reply:
Fixed this, let me know what you think of the new solution.